components/04-reference-pages/07-faculty-profile-page/faculty-profile-page.hbs

```diff
@@ -28,7 +28,7 @@
             Ph.D. Program Director, Professor of Accounting
           </h5>
         </div>
-        <div class="accounting-faculty-img-cntnt">
```

Comment:
this class name seems specific and doesn't have any styling (that I could find), can we remove entirely?